pkg/extensions/main.go

```diff
@@ -5,6 +5,7 @@ import (
 	// remove the cron (v1) dependency
 
 	_ "github.com/beevik/etree"
+	_ "github.com/cortexproject/cortex/pkg/util"
```

Comment:
Could you explain this addition, please?

Reply:
It's being used [here](https://github.com/grafana/grafana-enterprise/blob/main/src/pkg/extensions/auditing/loggers/loki_http.go#L7-L8). 

So, the underscore import is the way to force that library to be included in Go Modules metadata (`go.mod`/`go.sum`) with no direct usages from Grafana OSS. That's basically the reason of most of the underscore imports present here.

However, going back to this specific import, I see now it's being used at [ngalert](https://github.com/grafana/grafana/blob/main/pkg/services/ngalert/api/promql_compat.go#L10). That could allow us to remove this underscore import, but I'm not sure if that's a safe operation because we could stop using that package in the OSS code base at some point, then it won't be easy to figure out back that needed.

Is that explanation / context enough? 🤔

Reply:
Yes, thank you. We are trying to break the dependencies on Cortex to avoid version clashes.

`Backoff` is moving to here: https://github.com/grafana/dskit/blob/main/backoff/backoff.go
Similarly `flagext`.

`TimeFromMillis` is so trivial it should be copied and not reused.